### PR TITLE
doc updates for NLBs

### PR DESCRIPTION
**Pull request by ahl**
Reasoning for docs update: docs are a little inconsistent around the (relatively new) NLBs
 Relevant Terraform version: ready for immediate release

### CODE REVIEW

File: website/docs/r/ecs_service.html.markdown [hunk 1]
```diff
@@ -69,7 +69,7 @@ into consideration during task placement. The maximum number of
 Load balancers support the following:
 
 * `elb_name` - (Required for ELB Classic) The name of the ELB (Classic) to associate with the service.
-* `target_group_arn` - (Required for ALB) The ARN of the ALB target group to associate with the service.
+* `target_group_arn` - (Required for modern Load Balancers) The ARN of the Load Balancer target group to associate with the service.
```
Comment (bflad) [inline website/docs/r/ecs_service.html.markdown]: Can you change this to `(Required for ALB/NLB)` please? Amazon's definition of "modern" might change in the future.

Comment (ahl) [reply website/docs/r/ecs_service.html.markdown]: You got it!

File: website/docs/r/lb.html.markdown [hunk 2]
```diff
@@ -38,7 +39,24 @@ resource "aws_lb" "test" {
...
 # Create a new network load balancer
+resource "aws_lb" "test" {
+  name               = "test-lb-tf"
+  internal           = false
+  load_balancer_type = "network"
+  subnets            = ["${aws_subnet.public.*.id}"]
+
+  enable_deletion_protection = true
 
+  access_logs {
```
Comment (arohter) [inline website/docs/r/lb.html.markdown]: I don't think NLBs support access logs.

Comment (bflad) [reply website/docs/r/lb.html.markdown]: They do not, this configuration should be removed

Comment (ahl) [reply website/docs/r/lb.html.markdown]: fixed

**Pull request merged**